### PR TITLE
Resnet50

### PR DESCRIPTION
**Pull request by kjatkows**
fixed blank output in --log mode

### CODE REVIEW

File: kernels/module.cl [hunk 1]
```diff
@@ -26,20 +29,27 @@ kernel void heavy(const global char *input, global char *output)
...
+    for (int i = 1; i <= counter; i++)
     {
-        a = i * a * a;
+        a += i + i;
+        //a = a % 100;
 
     }
-    output[0] = a;
+    output[id] = a;
+    //if (id == 0)
```
Comment (tponieck) [inline kernels/module.cl]: remove commented code

File: kernels/module.cl [hunk 2]
```diff
@@ -26,20 +29,27 @@ kernel void heavy(const global char *input, global char *output)
...
 
 kernel void mem_bound_kernel(const global char *input, const global char *input2, global char *output, int counter)
 {
+    const size_t id = get_global_id(0);
     for (int i = 0; i < counter; i++)
     {
-        output[i] = input[i] + input2[i];
+        output[id] += input[id] + input2[id];
     }
+    //if (id == 0)
```
Comment (tponieck) [inline kernels/module.cl]: same here

File: src/zenon.cpp [hunk 3]
```diff
@@ -420,61 +425,65 @@ void zenon::create_cmd_list()
...
+            submit_kernel_to_cmd_list(cmp_bound_kernel, { input1_buffer, input2_buffer }, im_buf3, kernel_ts_event[50], { &kernel_ts_event[49] }, 1, 27278);                              //<-res5b_branch2c
+            submit_kernel_to_cmd_list(cmp_bound_kernel, { input1_buffer, input2_buffer }, im_buf4, kernel_ts_event[51], { &kernel_ts_event[50]}, 1, 29580);                               //<-res5c_branch2a
+            submit_kernel_to_cmd_list(cmp_bound_kernel, { input1_buffer, input2_buffer }, im_buf4, kernel_ts_event[52], { &kernel_ts_event[51] }, 1, 55398);                              //<-res5c_branch2b
+            submit_kernel_to_cmd_list(cmp_bound_kernel, { input1_buffer, input2_buffer }, im_buf6, kernel_ts_event[53], { &kernel_ts_event[52] }, 1, 27278);                              //<-res5c_branch2c
            // submit_kernel_to_cmd_list(mem_bound_kernel, { input1_buffer, input2_buffer }, im_buf1, kernel_ts_event[54], { &kernel_ts_event[53] }, 1, 1);                                //<-fc1000
```
Comment (tponieck) [inline src/zenon.cpp:487]: here as well

File: kernels/module.cl (hunk 1 above)
Comment (kjatkows) [reply kernels/module.cl]: @tponieck  done

**Review by tponieck — APPROVED** [review]

**Pull request merged**